src/main/java/seedu/address/model/event/UidList.java

```diff
@@ -0,0 +1,105 @@
+package seedu.address.model.event;
```

Comment:
From previous comment in Event.java, the method to give a list of Persons can be from UidList as well

Reply:
I'll implement the `getPersons(Model model)` method in UidList class, you can use `event.getUids().getPersons(Model model)` to get the list of persons from an event.

Reply:
thanks!